packages/core/src/env.pipe.ts

```diff
@@ -2,10 +2,11 @@ import { Pipe, PipeTransform } from '@angular/core';
 
 @Pipe({
   name: 'env',
+  pure: true,
```

Comment:
Pipes are pure by default :)

Reply:
Doing it explicitly prevents the change in the future. It's a passive declaration tbh